test/e2e/network/dns.go

```diff
@@ -379,21 +450,49 @@ var _ = SIGDescribe("DNS", func() {
 				CaptureStdout: true,
 				CaptureStderr: true,
 			})
-			if err != nil {
-				framework.Logf("Failed to execute dig command, stdout:%v, stderr: %v, err: %v", stdout, stderr, err)
-				return false, nil
+			Expect(err).NotTo(HaveOccurred(), "failed to examine resolv,conf file on pod, stdout: %v, stderr: %v, err: %v", stdout, stderr, err)
+			if !strings.Contains(stdout, "ndots:2") {
+				framework.Failf("customized DNS options not found in resolv.conf, got: %s", stdout)
 			}
-			res := strings.Split(stdout, "\n")
-			if len(res) != 1 || res[0] != testInjectedIP {
-				framework.Logf("Expect command `%v` to return %s, got: %v", cmd, testInjectedIP, res)
-				return false, nil
+
+			By("Verifying customized name server and search path are working...")
+			// Do dig on not-exist-dns-name and see if the injected DNS record is returned.
+			// This verifies both:
+			// - Custom search path is appended.
+			// - DNS query is sent to the specified server.
+			cmd = []string{"/usr/bin/dig", "+short", "+search", testDNSNameShort}
+			digFunc := func() (bool, error) {
+				stdout, stderr, err := f.ExecWithOptions(framework.ExecOptions{
+					Command:       cmd,
+					Namespace:     f.Namespace.Name,
+					PodName:       testUtilsPod.Name,
+					ContainerName: "util",
+					CaptureStdout: true,
+					CaptureStderr: true,
+				})
+				if err != nil {
+					framework.Logf("Failed to execute dig command, stdout:%v, stderr: %v, err: %v", stdout, stderr, err)
+					return false, nil
+				}
+				res := strings.Split(stdout, "\n")
+				if len(res) != 1 || res[0] != testInjectedIP {
+					framework.Logf("Expect command `%v` to return %s, got: %v", cmd, testInjectedIP, res)
+					return false, nil
+				}
+				return true, nil
 			}
-			return true, nil
-		}
-		err = wait.PollImmediate(5*time.Second, 3*time.Minute, digFunc)
-		Expect(err).NotTo(HaveOccurred(), "failed to verify customized name server and search path")
+			err = wait.PollImmediate(5*time.Second, 3*time.Minute, digFunc)
+			Expect(err).NotTo(HaveOccurred(), "failed to verify customized name server and search path")
 
-		// TODO: Add more test cases for other DNSPolicies.
-	})
+			// TODO: Add more test cases for other DNSPolicies.
+		})
+	}
+}
+
+var _ = SIGDescribe("DNS [IPv4]", func() {
```

Comment:
Feature:Networking-IPv4?

Reply:
here we cannot set [Feature:Networking-IPv4], because "Feature" is a keyword used in the e2e and most often just skipped.
We want to flag IPv4 - but keep the usual information, especially "Conformance".